bin/sidecli.re

```diff
@@ -806,6 +807,104 @@ let self = {
+  open Networking.Trusted_validators_membership_change;
+  let file = folder ++ "/identity.json";
+  let.await identity = Files.Identity.read(~file);
+  let payload = Trusted_validators_membership_change.{address, action: Add};
+  let payload_json_str =
+    payload
+    |> Trusted_validators_membership_change.to_yojson
+    |> Yojson.Safe.to_string;
+  switch (payload_json_str |> BLAKE2B.of_string) {
+  | None =>
+    await(`Error((false, "Internal Error: Failed to compute BLAKE2B")))
+  | Some(payload_hash) =>
+    let signature = Signature.sign(~key=identity.key, payload_hash);
+    let.await () =
+      Networking.request_trusted_validator_membership(
+        {signature, payload},
+        identity.uri,
+      );
+    await(`Ok());
+  };
+};
+let remove_trusted_validator = (folder, address) => {
+  open Networking.Trusted_validators_membership_change;
+  let file = folder ++ "/identity.json";
+  let.await identity = Files.Identity.read(~file);
+  let payload =
+    Trusted_validators_membership_change.{address, action: Remove};
+  let payload_json_str =
+    payload
+    |> Trusted_validators_membership_change.to_yojson
+    |> Yojson.Safe.to_string;
+  switch (payload_json_str |> BLAKE2B.of_string) {
+  | None =>
+    await(`Error((false, "Internal Error: Failed to compute BLAKE2B")))
+  | Some(payload_hash) =>
+    let signature = Signature.sign(~key=identity.key, payload_hash);
+    let.await () =
+      Networking.request_trusted_validator_membership(
+        {signature, payload},
+        identity.uri,
+      );
+    await(`Ok());
+  };
+};
+
+let info_add_trusted_validator = {
+  let doc = "Helps node operators maintain a list of trusted validators they verified off-chain which can later be used to make sure only trusted validators are added as new validators in the network.";
+  Term.info(
+    "add-trusted-validator",
+    ~version="%‌%VERSION%%",
+    ~doc,
+    ~exits,
+    ~man,
+  );
+};
+
+let info_remove_trusted_validator = {
+  let doc = "Helps node operators maintain a list of trusted validators they verified off-chain which can later be used to make sure only trusted validators are added as new validators in the network.";
+  Term.info(
+    "remove-trusted-validator",
+    ~version="%‌%VERSION%%",
+    ~doc,
+    ~exits,
+    ~man,
+  );
+};
+
+let address_t = {
+  let parser = string =>
+    string
+    |> Protocol.Address.of_string
+    |> Option.to_result(~none=`Msg("Expected a validator address."));
+  let printer = (fmt, address) =>
+    Format.fprintf(fmt, "%s", Protocol.Address.to_string(address));
+  Arg.(conv((parser, printer)));
+};
+
+let validator_address = {
+  let docv = "validator_address";
+  let doc = "The validator address to be added/removed as trusted";
+  Arg.(required & pos(2, some(address_t), None) & info([], ~docv, ~doc));
+};
+
+let add_trusted_validator = {
+  Term.(
+    lwt_ret(const(add_trusted_validator) $ folder_node $ validator_address)
+  );
+};
```

Comment:
Put them alongside the info declaration and also the code declaration, commands in this file have it's parts clustered.

Reply:
Addressed